profile.out

```diff
@@ -0,0 +1 @@
+mode: atomic
```

Comment:
What's this file?

Reply:
Hmmm have no idea, IDE junk probably :( will force remove it now